lib/shopify_transporter/pipeline/magento/order/top_level_attributes.rb

```diff
@@ -46,7 +46,7 @@ def financial_status(hash)
             order_state = hash['state']
             status = nil
 
-            if order_state == 'Pending Payment'
+            if order_state == 'Pending Payment' || unpaid?(hash)
```

Comment:
One side note: is it possible for an order to have total_paid = 0 and have refunds?

Reply:
hmm I doubt that's possible. There's nothing to refund for if the user has paid nothing in the first place.